nsxt/gateway_common.go

```diff
@@ -274,7 +274,7 @@ func initGatewayLocaleServices(d *schema.ResourceData, connector *client.RestCon
 
 	services := d.Get("locale_service").(*schema.Set).List()
```

Comment:
I think we should go into the update only if locale service changed (which is very rare occasion). This way we'll avoid the retry issue for most cases. I'm not sure HasChange("locale_service") would work for map (it didn't, but perhaps this was fixed with latest plugin sdk upgrade)
But GetChange definately works, and we can derive whether locale service is changed from there.
Perhaps this would be better as follow up patch.

Reply:
OK. I'll add a TODO, and do it later.

Reply:
ok. I'll add a TODO in the code